backend/tests/test_project.py

```diff
@@ -1,7 +1,7 @@
 import json
 import unittest
 import uuid
-from flask import current_app
+from flask import current_app, request
```

Comment:
@peterghrong Do you have an auto-import type of thing lol? This is unused I think.

Reply:
Honestly possibly at this point with all these dumb errors